lib/iris/fileformats/grib/__init__.py

```diff
@@ -48,9 +48,9 @@
 import iris.fileformats.grib.load_rules
 
 
-__all__ = ['as_messages', 'as_pairs', 'grib_generator', 'load_cubes',
-           'reset_load_rules', 'save_grib2', 'save_messages', 'GribWrapper',
-           'hindcast_workaround']
+__all__ = ['as_messages', 'load_pairs_from_fields', 'as_pairs',
```

Comment:
I notice that this controls the order of the docs, as well as content.
I'm good with the chosen contents, but I think it they could be in a more helpful order now.
Suggested:
 * load_cubes
 * save_grib2
 * load_pairs_from_fields
 * save_pairs_from_cube
 * as_messages
 * save_messages
 * GribWrapper  (deprecated)
 * as_pairs (deprecated)
 * grib_generator (deprecated)
 * reset_load_rules  [[Deprecated]]
 * hindcast_workaround  [[Deprecated]]